tests/integration/features/command/update.feature

```diff
@@ -0,0 +1,25 @@
+Feature: command/update
+
+  Background:
+    Given user "participant1" exists
+
+  Scenario: Create a public room with message expiration time and update removing the expiration time
+    Given invoking occ with "talk:room:create room1 --user participant1 --owner participant1 --public --message-expiration=3"
+    And user "participant1" is participant of the following rooms (v4)
+      | name  |
+      | room1 |
```

Comment:
Maybe we check here that setting the expiration worked?
Same after the update.

Because otherwise if the commands ever break, you start deebugging the failing expiration instead of checking if expiration it self actually works.

Also since the expiration is tested elsewhere (is it? Should be in chat/something, right?) We can skip it here after checking that the value is correctly set in the conversations data returned by this request

Reply:
> We can skip it here after checking that the value is correctly set in the conversations data returned by this request
Good!